modules/administration/pages/repo-metadata.adoc

```diff
@@ -11,124 +11,84 @@ include::_partials/entities.adoc[]
 endif::[]
 
 
-== Signing Repository Metadata
+TODO:: Explain why repository metadata should/would be signed.
 
-=== Generate a Custom GPG key
+You will require a custom GPG key to be able to sign repository metadata.
 
-To sign repository metadata a custom GPG key is required. Create a new GPG key as *root* via the following steps.
+.Procedure: Generating a custom GPG Key
 
+. As the root user, use the [command]``gpg`` command to generate a new key:
++
 ----
 $> gpg --gen-key
```

Comment:
I'd vote to get rid of the command-line indicator (`$>`) if this example does not print the command output.